api/src/server.rs

```diff
@@ -298,10 +389,28 @@ where
     pub async fn post_events(&self, event: Event) -> Result<EventsPostResponse, ErrorResponse> {
         let event_id = decode_event_id(&event.id)?;
         let event_data = decode_event_data(&event.data)?;
-        self.model
-            .insert(event_id, Some(event_data))
+        let (tx, rx) = tokio::sync::oneshot::channel();
+        tokio::time::timeout(
+            std::time::Duration::from_secs(3),
```

Comment:
Should we make this a constant? Why 3s here and 10s below?

Reply:
Yeah, good call on constants. I changed this to 1 second and left the other at 10. The names hopefully help clarify a bit, but I was thinking this should be short because enqueuing to the channel should be immediate, and if it's full we shouldn't wait that long and just fail the request. Once we accept the write, we are willing to wait longer to hear it request succeeded... In practice, 10s is pretty high, but I was trying to avoid the "I got an error but my write succeeded" case for js-ceramic without leaving an infinite timeout... I'm open to alternate values.